data/AnushaSP07.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Anusha SP",
+  "displayStatsPublic": true,
```

Comment:
As of our latest update, we no longer need the `displayStatsPublic` property. So please remove this property from your json. 

```suggestion
```